### PR TITLE
writeFile feature implementation

### PR DESCRIPTION
**Pull request by WaelYoussfi**

### CODE REVIEW

File: src/dev/flang/be/c/Intrinsics.java [hunk 1]
```diff
@@ -93,6 +93,12 @@ public class Intrinsics extends ANY
                                 outOrErr(in)
                               ));
         });
+    IntrinsicCode noFileIo = (c,cl,outer,in) ->
+      CStmnt.seq(CExpr.fprintfstderr("*** C backend support does not support fileio (yet).\n"),
```
Comment (fridis) [inline src/dev/flang/be/c/Intrinsics.java:97]: English sounds strange. Maybe remove the first `support`.

File: lib/fuzion/std/fileio.fz [hunk 2]
```diff
@@ -66,4 +66,26 @@ public fuzion.std.fileio is
                       # the length of the internal array representing the path
                       pathLength i32) i64 is intrinsic
 
-  private writeFile(path string, content array<u8>) is intrinsic # NYI
+  # creates a new file using an array of u8 as content and a string as the file path
+  # it overrides the file, if it exists, or creates a new one if it does not exist
```
Comment (fridis) [inline lib/fuzion/std/fileio.fz:70]: overrides -> overwrites?

**Review by fridis — COMMENTED** [review]
just nitpicking in the English text.

File: lib/fuzion/std/fileio.fz (hunk 2 above)
Comment (WaelYoussfi) [reply lib/fuzion/std/fileio.fz:70]: typo because i used to use @Override in Java

**Pull request merged**